lesson02/assignment/index.html

```diff
@@ -4,12 +4,45 @@
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link rel="stylesheet" href="css/style.css" />
+        <link rel="stylesheet" href="./css/style.css">
         <title>Assignment 02</title>
     </head>
     <body>
-        <main class="main">
+        <!-- <main class="main">
             <h1>title</h1>
-        </main>
+        </main> -->
+
+        <!-- Left side -->
+        <div class="card">
+            <div class="card-body">
+            <header class="left-side">
+                <img src="./img/headshot.jpg" class="card-img" alt="Paolo Maldini developer">
+                <h1 class="card-title">Paolo Maldini</h1>
+                <h2 class="card-subtitle">Front End Developer</h2>
+            </header>
+
+            <!-- Right Side -->
+            <!-- <article> -->
+            <div class="card-list right-side">
+                <ul class="list-group">
+                    <li class="list-group-item"><span>Company: </span>Web Weavers</li>
+                    <li class="list-group-item"><span>Experience: </span>3 years</li>
+                    <li class="list-group-item"><span>Schools: </span>University of Washington</li>
+                    <li class="list-group-item"><span>Major: </span>Marketing</li>
+                    <li class="list-group-item"><span>Email: </span><span class="email-list">paolo@example.com</span></li>
+                    <li class="list-group-item-link"><img src="./img/linkedin.svg" class="linkedin-img" alt="linkedin icon"><a class="linkedin-link" href="https://www.linkedin.com/"><span class="linkedin-list">paolo.linkedinprofile.com</span></a></li>
+                </ul>
+                <!-- <div class="card-footer"> -->
+                    <!-- <img src="./img/linkedin.svg" class="linkedin-img" alt="linkedin icon">
+                    <a class="linkedin-link" href="">paolo.linkedinprofile.com</a> -->
+                    <!-- <p class="linkedin-link"></p> -->
+                <!-- </div> -->
```

Comment:
No need to keep code that isn't being used.

Reply:
Thanks for letting me know. I just removed them!